### ISSUE TITLE
Primer acchurador de galaxias

### ISSUE THREAD

**Issue by leliel12** [vcristiani/galaxy-chop#82]
> Depende #81

- [x] Leer el detalle este documento sobre las API de scikit-learn: [API design for machine learning software:
experiences from the scikit-learn project](https://arxiv.org/pdf/1309.0238.pdf?source=post_elevate_sequence_page---------------------------)
- [x] Implementar en el modulo  `galaxychop.core`  la clase `GCDecomposeMixin` queue tiene exactamente esta forma

```python
class GCDecomposeMixin:

    def decompose(self, galaxy):
        if not isinstance(galaxy, core.Galaxy):
            found = type(galaxy)
            raise TypeError(
                f"'galaxy' must be a galaxychop.Galaxy instance. Found {found}")
            
        X = galaxy.values()
        return self.fit_transform(X)
```

- [x] Implementaar en el modulo `galaxychop.core` la clase abstracta `GCClusterMixi`n exactamente con esta format

```python
from sklearn.base import ClusterMixin
class GCClusterMixin(GCDecomposeMixin, ClusterMixin):
    pass
```

- [x] Implementar clase `GCKMeans`  que deberia heredar de `GCClusterMixin`

test

```python
def test_GCKmeans(galaxy):
    gckmeans = GCKmeans(components=5, random_state=0)
    result = gckmeans.decompose(galaxy)
    
    kmeans = KMeans(n_clusters=5, random_state=0)
    expected = kmeans.fit_transform(galaxy.values())
    
    np.testing.assert_array_equals(result, expected)
```

**Comment by vcristiani** [comment]
I did the implementation of the GCKmeans class, but I don't know how to change the name of the n_clusters parameter, which it inherits from KMeans, to n_components.

**Comment by vcristiani** [comment]
this closes with #101
